packages/server/package.json

```diff
@@ -4,13 +4,13 @@
   "private": true,
   "main": "index.js",
   "scripts": {
-    "build-prod": "tsc || echo 'built, with type errors'",
-    "check-ts": "tsc --noEmit",
+    "build-prod": "yarn ensure-cloud-validations && tsc || echo 'built, with type errors'",
+    "check-ts": "yarn ensure-cloud-validations && tsc --noEmit",
     "clean-deps": "rimraf node_modules",
     "codecov": "codecov",
     "dev": "node index.js",
     "docker": "cd ../.. && WORKING_DIR=/packages/server ./scripts/run-docker-local.sh",
-    "postinstall": "patch-package",
+    "postinstall": "patch-package && yarn sync-cloud-validations",
```

Comment:
if we are doing this at the package level, do we still need to call it in CI after the build? Would be nice to get rid of this https://github.com/cypress-io/cypress/blob/develop/.circleci/src/pipeline/@pipeline.yml#L201